stable/rocketchat/OWNERS

```diff
@@ -1,4 +1,6 @@
 approvers:
 - verwilst
+- srueg
```

Comment:
Does this mean you will keep on developing/maintaining this chart, or was this a one-time effort?

Reply:
Yes, I'll keep maintaining it. If that's ok with you.